### PR TITLE
Adding clarification to 'first step' guide

### PR DESCRIPTION
**Pull request by ShelbyZ**
## Proposed changes

Running through the initial setup guide presented a problem that I did not have FireFox installed on my test system.  It seems like there were two options either installing FireFox or finding a web driver.  This felt like a step that could be mentioned in the initial documentation as it would effect some users of WebdriverIO.

## Types of changes

What types of changes does your code introduce to WebdriverIO?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### Reviewers: @christian-bromann

Guide expects user has FireFox installed which may not be correct in all cases and present the user with error information in their command prompt running selenium.  While that error information is useful enough to point out that selenium cannot find the binary path, it does create a disconnect between new users of WebdriverIO.

### CODE REVIEW

File: docs/guide.md [hunk 1]
```diff
@@ -56,7 +56,11 @@ webdriverio
     .end();
 ```
 
-** 6. Run your test file**
+** 6. Install browser and appropriate driver**
+
+In the example above FireFox is specified as the desired browser, but may not be currently installed on the testing system. The choice is to either install FireFox or download [Third Party Drivers](http://docs.seleniumhq.org/download/#thirdPartyDrivers) and place them in the same directory as the selenium standalone server.
```
Comment (christian-bromann) [inline docs/guide.md]: > and place them in the same directory as the selenium standalone server.

What does this mean? Just put the binary in the same directory as your jar file? I don't thin that is how it works

Comment (ShelbyZ) [inline docs/guide.md]: Well there are two options, they could install the drivers in a globally available path location or local to the selenium jar file.  The only documentation I could quickly find that references adding the path to drivers was in FireFox's documentation around their Marionette executable - [Setting up the Marionette executable](https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette/WebDriver#Setting_up_the_Marionette_executable).

Comment (christian-bromann) [inline docs/guide.md]: But you say: _"[...] to either install Firefox or install Third Party Drivers [...]"_ which is not quite right. If you install Firefox at the default path you should be find by running a selenium standalone server. However if you only want to run a driver you need to start the driver server. I just don't get what you mean by this sentence.

Comment (ShelbyZ) [inline docs/guide.md]: I suppose there are two ways of looking at it either selenium standalone server can load the web driver or the driver can be run standalone.  If the goal of the guide is to have users run with FireFox then they are required to have it installed.  In the other case, assuming they want to utilize WebdriverIO, they should download a web driver and have selenium standalone server take care of running it.

Comment (christian-bromann) [inline docs/guide.md]: Yes, so if you want to run a Firefox test you should have Firefox installed on your system.

**Comment by christian-bromann** [comment]
Sorry but I don't think this is a good addition to the docs, it rather complicates things. Thanks though!

**Pull request closed without merging**